### PR TITLE
Retention graph and multiple event fix

### PR DESCRIPTION
**Pull request by EDsCODE**
## Changes

*Please describe.*  
*If this affects the frontend, include screenshots.*  

## Checklist

- [ ] All querysets/queries filter by Organization, by Team, and by User
- [ ] Django backend tests
- [ ] Jest frontend tests
- [ ] Cypress end-to-end tests

### CODE REVIEW

File: frontend/src/lib/components/ChartFilter/chartFilterLogic.js [hunk 1]
```diff
@@ -28,9 +29,11 @@ export const chartFilterLogic = kea({
         },
     }),
     urlToAction: ({ actions }) => ({
-        '/insights': (_, { display }) => {
+        '/insights': (_, { display, insight }) => {
```
Comment (EDsCODE) [inline frontend/src/lib/components/ChartFilter/chartFilterLogic.js:32]: @timgl Fixed! Rebasing master dropped this parameter accidentally

**Comment by EDsCODE** [comment]
@timgl was there anything else I should address with this? Otherwise, I'm going to test a bit more and merge because it's blocking for most of the other tasks

**Comment by paolodamico** [comment]
I think it might be worth adding a PR description and maybe some screenshots for posterity.

**Comment by timgl** [comment]
Yeah I think there were other issues (looks like you reversed again). Let me test this properly tomorrow

**Pull request merged**